map/framework.cpp

```diff
@@ -2712,11 +2735,18 @@ osm::Editor::SaveResult Framework::SaveEditedMapObject(osm::EditableMapObject em
       shouldNotify = true;
     }
 
+    // TODO(mgsergio): Consider this:
+    // A user changed a feature with no original addres and addres differ from the one
+    // on a hosting building. He saved it. This was uploaded to OSM. Then he set
+    // back an address from a hosting building. As a result this address woun't
```

Comment:
will not be merged into OSM, because we clear street and house number in the following code.